README.md

```diff
@@ -27,6 +27,19 @@ Quick note, it is considered a good practice to use Google before contacting men
 
 For any troubles you're having with the organization, mentors or other students, feel free to contact the GCI admins (details below).
 
+## Don’t do these things
+- Don't use foul language to any student or mentor in any case. Always be polite to other community members and mentors.
+- You should never argue with mentors with regarding your work . They are highly experienced people if they ask you to do some change   you should follow their advice.
```

Comment:
I think this line could be written "softer". Sounds too harsh.

Reply:
Okay I will make the changes asap

Reply:
I have made the changes.